android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/responses/GetVehicleDataResponseTests.java

```diff
@@ -202,6 +202,9 @@ public void testJson() {
             bodyInformationObj.put(BodyInformation.KEY_IGNITION_STABLE_STATUS, VehicleDataHelper.BODY_INFORMATION_IGNITION_STATUS);
```

Comment:
@kostyaBoss I think you can revert these changes, as vehicle data does not need to be changed for this PR.  The corresponding PR for the JavaScript suite (https://github.com/smartdevicelink/sdl_javascript_suite/pull/354/files) did not make changes to the vehicle data.